samples/subsys/usb/uvc/src/main.c

```diff
@@ -73,20 +73,62 @@ static void app_add_format(uint32_t pixfmt, uint16_t width, uint16_t height, boo
 	uvc_add_format(uvc_dev, &fmt);
 }
 
+struct video_resolution {
+	uint16_t width;
+	uint16_t height;
+};
+
+static struct video_resolution video_common_fmts[] = {
+	{ .width = 3840,	.height = 2160,	},	/* UHD */
+	{ .width = 1920,	.height = 1080,	},	/* FHD */
+	{ .width = 1280,	.height = 1024,	},	/* SXGA */
+	{ .width = 1280,	.height = 720,	},	/* HD */
+	{ .width = 800,		.height = 600,	},	/* SVGA */
+	{ .width = 854,		.height = 480,	},	/* WVGA */
+	{ .width = 640,		.height = 480,	},	/* VGA */
+	{ .width = 320,		.height = 240,	},	/* QVGA */
+	{ .width = 160,		.height = 120,	},	/* QQVGA */
```

Comment:
The default is 5, how many "constrained" devices that Zephyr aims to support can handle UHD, FHD, SXGA, HD, and SVGA and on what frame rate? Should it be sorted ascending?